website/content/docs/nia/terraform-modules.mdx

```diff
@@ -0,0 +1,123 @@
+---
+layout: docs
+page_title: Compatible Terraform Modules for NIA
```

Comment:
I like having a modules-specific page! I think it will make it easier for users to have all the module info in a single place